plugins/importtext_procedure.cc

```diff
@@ -56,6 +56,12 @@ ImportTextConfigDescription::ImportTextConfigDescription()
     addField("ignoreBadLines", &ImportTextConfig::ignoreBadLines,
              "If true, any line causing a parsing error will be skipped. "
              "Empty lines are considered bad lines.", false);
+    addField("structuredNames", &ImportTextConfig::structuredNames,
```

Comment:
I thought this would apply to row names as well but clearly it's just column names, so can we actually name this option `structuredColumnNames` please? (sorry)